Makefile

```diff
@@ -78,15 +78,15 @@ isort-check:
 yamllint:
 	yamllint .
 
-.PHONY: check-version
-check-version:
-	bash ./extras/check_version.sh $(VERSION)
```

Comment:
I think we will need to adapt the `.github/workflows/docker.yml`, because we used this arg there.

Since you ported it to be an env var, we should set it as an env var there instead.

Reply:
Oh, right, I'll fix it.

Reply:
Done. I've fixed it and tested with `act` with a wrong and correct tag and both work as expected.